pkg/util/timeconv/duration/duration.go

```diff
@@ -0,0 +1,161 @@
+package duration
+
+import (
+	"strconv"
+	"time"
+)
+
+type DurationUnit string
```

Comment:
for package-level elements, order them by   imports, consts, vars, types, funcs.  So move this defnition down below the consts (even if it's referenced in consts)